tests/test_velocity_model_plots.py

```diff
@@ -0,0 +1,106 @@
+import subprocess
+import tempfile
+from pathlib import Path
+
+import diffimg
+import pytest
+import typer
+
+from velocity_modelling.scripts import plot_velocity_model
+
+PLOT_IMAGE_DIRECTORY = Path("wiki/images")
+INPUT_FFP = Path(__file__).parent / "input"
+
+SWEDGE_RHO3D_FILE_LINK = "https://www.dropbox.com/scl/fi/y8qf8zjq4zf9yeejsopya/rho3dfile.d?rlkey=87xt4kpac94dwf9hfo5pjepyb&st=n0x5qvit&dl=0"
```

Comment:
Leave the actual dropbox path in case the data file needs an update in the future

Reply:
Sure thing, I'll include it as a comment above.